$OpenBSD: patch-src_statshandler_cpp,v 1.1 2013/08/30 22:42:04 sthen Exp $

gloox's overloaded addAttribute() function doesn't have a variant to
handle long long. This value only holds the machine's uptime anyway so
int is sufficient.

--- src/statshandler.cpp.orig	Thu Aug 29 14:46:49 2013
+++ src/statshandler.cpp	Thu Aug 29 14:46:55 2013
@@ -172,7 +172,7 @@ Tag* GlooxStatsHandler::handleTag (Tag *stanzaTag){
 				t = new Tag("stat");
 				t->addAttribute("name","uptime");
 				t->addAttribute("units","seconds");
-				t->addAttribute("value",seconds - m_startTime);
+				t->addAttribute("value",(int)(seconds - m_startTime));
 				query->addChild(t);
 			} else if (name == "users/registered") {
 				t = new Tag("stat");
